app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="forgot_password">Quên mật khẩu?</string>
+    <string name="login">Đăng nhập</string>
+    <string name="sign_up">Đăng ký</string>
+    <string name="font_serif">serif</string>
+    <string name="do_not_have_an_account">Bạn chưa có tài khoản?</string>
+    <string name="where_are_you">Bạn ở đâu?</string>
+    <string name="select_domain">Chọn miền:</string>
+    <string name="next">Tiếp tục</string>
+    <string name="welcome">Chào mừng đến với</string>
+    <string name="btn_detail_profile">Thông tin cá nhân</string>
+    <string name="order_history">Lịch sử đặt hàng</string>
+    <string name="order_management">Quản lý đặt hàng</string>
+    <string name="shop_management">Quản lý cửa hàng</string>
+    <string name="domain_management">Quản lý miền</string>
+    <string name="logout">Đăng xuất</string>
+    <string name="featured">Tính năng</string>
+    <string name="new_">Mới</string>
+    <string name="sale_off">Giảm giá</string>
+    <string name="email">Email: </string>
+    <string name="chatwork_id">Mã Chatwork: </string>
+    <string name="join_date">Ngày tham gia: </string>
+    <string name="status">Trạng thái: </string>
+    <string name="description">Mô tả: </string>
+    <string name="new_password">Mật khẩu mới</string>
+    <string name="confirm_new_password">Xác nhận mật khẩu mới</string>
+    <string name="current_password">Mật khẩu hiện tại</string>
+    <string name="submit">Xác nhận</string>
+    <string name="money_unit">\u0020VNĐ</string>
+    <string name="login_error">Lỗi đăng nhập !</string>
+    <string name="login_success">Đăng nhập thành công !</string>
+    <string name="wrong_email_or_password">Không thể tìm thấy người dùng!</string>
+    <string name="message">Tin nhắn</string>
+    <string name="ok">OK</string>
+    <string name="cannot_load_the_data">Không thể tải dữ liệu</string>
+    <string name="email_is_Empty">Email không được để trống !</string>
+    <string name="password_is_Empty">Mật khẩu không được để trống!</string>
+    <string name="you_need_to_choose_a_domain_to_continue">Bạn cần chọn miền để tiếp tục!</string>
+    <string name="none">Không</string>
+    <string name="search">Tìm kiếm...</string>
+    <string name="product">Sản phẩm</string>
+    <string name="shop">Cửa hàng</string>
+    <string name="please_click_back_again_to_exit">Làm ơn bấm Back lần nữa để thoát!</string>
+    <string name="confirm_new_password_does_not_match">Xác nhận mật khẩu mới không trùng khớp</string>
+    <string name="current_password_is_empty">Mật khẩu hiện tại để trống</string>
+    <string name="update_profile_success">Cập nhật thông tin cá nhân thành công</string>
+    <string name="nothing_change">Không có gì thay đổi</string>
+    <string name="change_password">Thay đổi mật khẩu</string>
+    <string name="change_domain">Thay đổi miền</string>
+    <string name="shop_name">Tên cửa hàng: </string>
+    <string name="product_quantity">Số lượng sản phẩm: </string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.product_quantity` appears to be unused
 Unused resources make applications larger and slow down builds.